Nightly Remix Test fails: dxvk-remix moved OpenUSD to an internal packman remote

## Issue Description

The `Nightly Remix Test` workflow has failed since 2026-08-19. The job `rtx-remix-shader-test` aborts in the *Download external dependencies (packman)* step because a dxvk-remix dependency moved to NVIDIA's internal NVGTL packman remote, which requires a token we cannot use in public CI.

This is not a Slang regression — no Slang change is involved, and the shader build never starts.

## Reproduction

```
.github/workflows/nightly-remix-test.yml -> step "Download external dependencies (packman)"
cd dxvk-remix && ./scripts-common/packman/packman.cmd pull packman-external.xml
```

## Actual Behavior

```
Package 'open_usd' at version '25.11+07744417-windows-x86_64-release-nopython' is missing from local storage.
packman(ERROR): The environment variable 'NVM_GTLAPI_TOKEN' must be set to use NVGTL
##[error]Process completed with exit code 1.
```

Failing run: https://github.com/shader-slang/slang/actions/runs/32211785332 (11 consecutive green nights before it).

## Root Cause

dxvk-remix commit [`673dc2af`](https://github.com/NVIDIAGameWorks/dxvk-remix/commit/673dc2af) ("[HdRemix] Use USD without Python", 2026-08-18 22:07 UTC) replaced the OpenUSD dependency in `packman-external.xml`:

```diff
-<package name="usd.py311.windows-x86_64.stock.debug"   version="0.25.11-gl.18041+v25.11.363a7c8d" />
-<package name="usd.py311.windows-x86_64.stock.release" version="0.25.11-gl.18041+v25.11.363a7c8d" />
-<package name="python" version="3.11.10+nv1-windows-x86_64" />
+<package name="open_usd" version="25.11+07744417-windows-x86_64-release-nopython" />
+<package name="open_usd" version="25.11+07744417-windows-x86_64-debug-nopython" />
```

`packman` resolves remotes in the order `cloudfront -> rtx-remix-external -> gtl` (`scripts-common/packman/config.packman.xml`). The old `usd.py311.*` packages are on the public CloudFront CDN; `open_usd` is on neither public remote, so packman falls through to `gtl` and demands the token. All four flavours probed (`release`/`debug`, with and without `-nopython`) return 403 from the public CDN, so the whole package family appears to have no public presence.

Our nightly picked this up immediately because the clone is not pinned, despite the step comment claiming it is:

```yaml
# Clone dxvk-remix with all submodules (pinned to specific commit for stability)
git clone --recursive --depth 1 https://github.com/NVIDIAGameWorks/dxvk-remix.git
```

This appears to be an upstream publishing gap rather than a deliberate decision: dxvk-remix's own public CI broke 8 minutes after that commit ([run 32191742431](https://github.com/NVIDIAGameWorks/dxvk-remix/actions/runs/32191742431), same `packman pull` via `meson.build:48`), and their manifest tags only the *debug* USD entry `non-redist`, implying the release build is meant to be publicly redistributable.

## Options

Setting `NVM_GTLAPI_TOKEN` is out of scope — an NVIDIA-internal registry credential must not be placed in public OSS CI.

1. Pin the dxvk-remix clone to `673dc2af~1` to restore the nightly, and make the existing comment true. Temporary; stops tracking Remix HEAD.
2. Build shaders without USD. USD is a C++ dependency of the Remix toolkit and HdRemix; no shader includes a USD header. If meson only needs the directory to exist at configure time, a stub would do — this is the same workaround already applied for `rtx-remix-ngx_sdk_dlfg`. Under investigation.
3. Report upstream and wait for a public `open_usd` mirror.

## Test Plan

- Re-run `Nightly Remix Test` and confirm `rtx-remix-shader-test` reaches the *Verify shader compilation* step.

--- a/.github/workflows/nightly-remix-test.yml
+++ b/.github/workflows/nightly-remix-test.yml
@@ -110,14 +110,32 @@ jobs:
             exit 1
           fi
 
-      # Clone dxvk-remix with all submodules (pinned to specific commit for stability)
+      # Clone dxvk-remix at a pinned commit.
+      #
+      # The pin is load-bearing, not just a reproducibility nicety.
+      # dxvk-remix 673dc2af ("[HdRemix] Use USD without Python") switched
+      # the OpenUSD dependency to the `open_usd` packman package, which is
+      # published only to NVIDIA's internal NVGTL remote. `packman pull`
+      # then fails with "the environment variable 'NVM_GTLAPI_TOKEN' must
+      # be set to use NVGTL", which public CI cannot and should not
+      # satisfy. a6c0b547 is the last commit this workflow ran green
+      # against. See shader-slang/slang#12617.
+      #
+      # To bump: pick a dxvk-remix commit whose packman-external.xml
+      # resolves entirely from the public remotes (cloudfront /
+      # rtx-remix-external), then re-run this workflow to confirm.
       - name: Clone dxvk-remix repository
+        env:
+          DXVK_REMIX_COMMIT: a6c0b547306a8d3c97630de58c27635173b12c9a
         run: |
-          echo "Cloning dxvk-remix repository..."
-          git clone --recursive --depth 1 https://github.com/NVIDIAGameWorks/dxvk-remix.git
+          echo "Cloning dxvk-remix at $DXVK_REMIX_COMMIT..."
+          git init -q dxvk-remix
           cd dxvk-remix
-          commit=$(git rev-parse HEAD)
-          echo "Using dxvk-remix commit: $commit"
+          git remote add origin https://github.com/NVIDIAGameWorks/dxvk-remix.git
+          git fetch -q --depth 1 origin "$DXVK_REMIX_COMMIT"
+          git checkout -q FETCH_HEAD
+          git submodule update --init --recursive --depth 1
+          echo "Using dxvk-remix commit: $(git rev-parse HEAD)"
 
       # Cache packman packages between runs to save ~2-5 minutes
       - name: Cache Packman packages

